controllers/rollingupgrade_controller.go

```diff
@@ -761,6 +802,12 @@ func (r *RollingUpgradeReconciler) UpdateInstance(ctx *context.Context,
 		return
 	}
 
+	err = r.WaitForTermination(nodeName, r.generatedClient.CoreV1().Nodes())
```

Comment:
Instead of getting all the nodes and filtering, can we simply Get the required node?

`r.generatedClient.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{})` should do it

Reply:
Yes, but in that case we will have to rely on NotFound error.
I will make the change

Reply:
Done